src/Record/Builder.purs

```diff
@@ -45,6 +46,10 @@ newtype Builder a b = Builder (a -> b)
 build :: forall r1 r2. Builder (Record r1) (Record r2) -> Record r1 -> Record r2
 build (Builder b) r1 = b (copyRecord r1)
 
+-- | Build a record from scratch.
+buildFromScratch :: forall r. Builder (Record ()) (Record r) -> (Record r)
```

Comment:
Oh oops sorry, another nit: the parens around the Record r at the end shouldn’t be necessary.

Reply:
Those parens are no more.